ops/terraform/services/pipeline/modules/bfd_pipeline_slis/lambda_src/dev-requirements.in

```diff
@@ -1,13 +1,8 @@
 # Yes, boto3 _is_ a dev-requirement as this Lambda is not a Docker-based Lambda. Python Lambdas
 # using the default runtime run in an environment that already has boto3 installed, so this is
 # declared explicitly for developers working on this Lambda locally
```

Comment:
Can we remove this comment? It's no longer valid now that boto3 was moved out of dev-requirements.

Reply:
removed